CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+### Fixed
+- Fix issue where in the FCC subtitle settings menu, two options were returning the same value
```

Comment:
```suggestion
## [development]

### Fixed
- FCC subtitle settings menu showing two options with the same value
```

Reply:
@wasp898 are you sure this is correct? We always used to have `develop` as section name (because the dev branch is also named `develop`).

Reply:
Ahh, then we started messing up at some point, sorry:
https://github.com/bitmovin/bitmovin-player-ui/pull/587/files
https://github.com/bitmovin/bitmovin-player-ui/pull/583/files